### PR TITLE
Fix for missing code in Grid Pagination storybook

### PR DESCRIPTION
**Pull request by bojan88**
Signed-off-by: Bojan Durdevic <bojan88@gmail.com>

#### What does this PR do?
The code is missing for Pagination -> Grid example. This PR is fixing the issue.

#### What are the relevant issues?
I couldn't fine any issues opened for this.

#### Screenshots (if appropriate)

Before:
image

---

After:
image

#### Do the grommet docs need to be updated?
No

#### Should this PR be mentioned in the release notes?
No

### CODE REVIEW

**Review by jcfilben — APPROVED** [review]

File: src/js/components/Pagination/stories/Grid.js [hunk 1]
```diff
@@ -17,7 +24,7 @@ const CardResult = ({ item }) => (
   </Card>
 );
 
-const PaginatedGrid = () => {
+export const Grid = () => {
```
Comment (ericsoderberghp) [inline src/js/components/Pagination/stories/Grid.js:27]: I think we should keep the use of `Grid` to refer to the grommet component, as that will work well if cutting and pasting parts of the code and be less confusing.

Comment (bojan88) [reply src/js/components/Pagination/stories/Grid.js:27]: That's a good point. I'll try to figure out what exactly is the problem here.
What do you think about renaming this story as a last resort solution?

Comment (ericsoderberghp) [reply src/js/components/Pagination/stories/Grid.js:27]: I think we can do something similar to this: https://github.com/grommet/grommet/blob/master/src/js/components/DropButton/stories/typescript/Calendar.tsx

Comment (bojan88) [reply src/js/components/Pagination/stories/Grid.js]: Thanks Eric. This works great.

**Review by ericsoderberghp — APPROVED** [review]

**Comment by ericsoderberghp** [comment]
Thanks for your contribution!

**Pull request merged**